eslint error

This is what I get when I try to commit any changes in `src/functions/`:

```
> usaco-guide@0.1.0 lint-staged
> lint-staged

[STARTED] Preparing...
[SUCCESS] Preparing...
[STARTED] Running tasks...
[STARTED] Running tasks for *.{js,ts,jsx,tsx}
[STARTED] Running tasks for *.{js,jsx,ts,tsx,json,md,mdx}
[STARTED] eslint
[STARTED] prettier --write
[SUCCESS] prettier --write
[SUCCESS] Running tasks for *.{js,jsx,ts,tsx,json,md,mdx}
[FAILED] eslint [FAILED]
[SUCCESS] Running tasks...
[STARTED] Applying modifications...
[SKIPPED] Skipped because of errors from tasks.
[STARTED] Reverting to original state because of errors...
[SUCCESS] Reverting to original state because of errors...
[STARTED] Cleaning up...
[SUCCESS] Cleaning up...

✖ eslint:

/Users/benq/Dev/usaco-guide/src/functions/src/submitContactForm.ts
  0:0  error  Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: src/functions/src/submitContactForm.ts.
The file must be included in at least one of the projects provided

✖ 1 problem (1 error, 0 warnings)
```

--- a/package.json
+++ b/package.json
@@ -176,7 +176,7 @@
     "url": "https://github.com/thecodingwizard/usaco-guide/issues"
   },
   "lint-staged": {
-    "*.{js,ts,jsx,tsx}": "eslint",
+    "*.{js,ts,jsx,tsx}": "eslint --config .eslintrc.js --resolve-plugins-relative-to .",
     "*.{js,jsx,ts,tsx,json,md,mdx}": "prettier --write"
   },
   "babelMacros": {

